src/read.jl

```diff
@@ -104,113 +104,160 @@ function read_storage(filepath)::Storage
 end
 
 
-"""Load demand flexibility profile from .csv files into DataFrame(s)."""
+"""
+    read_demand_flexibility(filepath, interval)
```

Comment:
Is it something we forget to delete here?

Reply:
This is the docstring convention for Julia functions.